### PR TITLE
Webviews

### PR DESCRIPTION
**Pull request by LucasKuhn**

### CODE REVIEW

**Comment by LucasKuhn** [comment]
@ambegossi 
Added an ETag option to the API Call, if we save the ETag from the last request, and nothing changed on the Github AP since the last request, Github API will repond with a status: 304 Not Modified and... **The request will not count for the limit of 60 requests!** 🎉

Not working yet, but I think it would be fun to implement:
https://developer.github.com/v3/#rate-limiting

** EDIT: **

Turns out that the Retrofit checks for this ETag, and deals with that, we just needed to add a cache:
https://futurestud.io/tutorials/retrofit-2-activate-response-caching-etag-last-modified

It is working 😊

**Comment by ambegossi** [comment]
@LucasKuhn 
aí tu deitou! :clap: :clap:

File: app/src/main/java/ucs/trabalho3/github_events/src/service/ApiRequestService.java [hunk 1]
```diff
@@ -72,44 +72,48 @@ public void run() {
...
-
-                            @Override
-                            public void onFailure(Call<List<Event>> call, Throwable t) {
-                                // Log error here since request failed
-                                Log.e("erro", t.toString());
-                            }
-                        });
-                    }
 
+                    CallApi();
```
Comment (ambegossi) [inline app/src/main/java/ucs/trabalho3/github_events/src/service/ApiRequestService.java:76]: much better now :ok_hand:

**Comment by ambegossi** [comment]
@LucasKuhn teste

**Pull request merged**